### PR TITLE
9_Distinct Echo Substrings (HASHING)

### PR DESCRIPTION
**Pull request by soma2000-lang**
#222 
Code writtten in C++

### CODE REVIEW

**Comment by soma2000-lang** [comment]
@Debashish-hub please check

File: Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                   
+           }
+        }
+        return s.size();//returning
+    }
+};
+int main(){
+string text;
+cin>>text;
+distinctEchoSubstrings(string text);
```
Comment (Debashish-hub) [inline Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp:33]: Write proper main function. You need to print it. @soma2000-lang

**Review by Debashish-hub — CHANGES_REQUESTED** [review]

**Comment by soma2000-lang** [comment]
@Debashish-hub  check now

**Comment by Debashish-hub** [comment]
> @Debashish-hub check now

Where are you printing the output?

**Comment by soma2000-lang** [comment]
@Debashish-hub  Sorry .done check now

File: Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
```
Comment (Debashish-hub) [inline Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp:33]: You are returning s.size() in the function. You just need to print distinctEchoSubstrings(string text) not s.size()
@soma2000-lang

**Review by Debashish-hub — CHANGES_REQUESTED** [review]

**Comment by soma2000-lang** [comment]
@Debashish-hub  actually in this question I was asked to print the  no of the distict echo substrings so in this code inserted all this distinct echoes in  s.So the size of s will be now my no of disctinct echoes so I am just calling this function and returning s.size()

**Comment by soma2000-lang** [comment]
@Debashish-hub  please check

File: Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
+//brute force approach
+//time complexity=0(n2)
+//space complexity 0(n)
+
+class Solution {
```
Comment (Debashish-hub) [inline Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp:5]: You are using class but not declaring class in main function. Either declare remove the class or declare class and obj in main. @soma2000-lang

**Review by Debashish-hub — CHANGES_REQUESTED** [review]

**Comment by soma2000-lang** [comment]
@Debashish-hub  please check now

**Comment by soma2000-lang** [comment]
@Debashish-hub why did u close this??

**Pull request merged**